public/js/redux/updateProgressRedux.js

```diff
@@ -0,0 +1,11 @@
+import { State } from 'jumpstate';
+
+const updateProgressRedux = State({
+    initial: false,
+
+    toggleIsUpdatingCharts(state, isUpdating) {
```

Comment:
This is probably be a refactoring task after a merge and happy to discuss if you disagree, but it might be nice if reducers could be:

```javascript
const toggleIsUpdatingCharts = (state, isUpdating) => isUpdating;
const updateProgressRedux = State({ initial: false, toggleIsUpdatingCharts });
```

Then adding more over time would save a lot of curly braces and return statements!